src/components/Layout.vue

```diff
@@ -96,18 +83,15 @@
 
 <script setup>
 import { ref, onMounted, onUnmounted } from 'vue';
+import Prose from '../components/Prose.vue';
+
+const props = defineProps({
+  collectionName: { type: String, required: true },
+  headings: { type: Object, required: true },
```

Comment:
Headings is provided as a flat structure in Astro - this need to the removal of the nested HTML above